challenge-6/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  </head>
+    <input type="text" id="number"></input>
+    <button id="test">Test</button>
+  <body>
+    <script>
+      $('#test').click(() => {
+        let response = $('#number').val();
+        response = response.split('').map(Number);
```

Comment:
😱 👍 very good technique. Never even know you could do this!

I would actually make line 11 a constant and name the variable something like `const creditCardNumber`. Then make line 12 a const as well and name it something like numbers or digits. The idea is that the variables should be very clear what they do.

In this case, I think 2 `const` is better since a `let` means the variable can continue to be modified. When you write a lot of code, another person might wonder if the variable is modified later in the code.